README.md

```diff
@@ -1,10 +1,10 @@
-<p align="center"><a href="https://github.com/Zuehlke"><img src="https://raw.githubusercontent.com/webplatformz/zuehlke-reveal-package/feature/new-slidedeck/src/assets/img/zuehlke-logo.jpg" alt="Zuehlke Logo" height="90"/></a></p>
+<p align="center"><a href="https://github.com/Zuehlke"><img src="https://raw.githubusercontent.com/romanutti/zuehlke-reveal-package/feature/new-slidedeck/src/assets/img/zuehlke-logo.jpg" alt="Zuehlke Logo" height="90"/></a></p>
```

Comment:
I guess this should not be changed for the pr towards the original webplatformz repo (valid for all the changed in the README.md). Time to get rid of my fork so that there is only one source of truth

Reply:
reverted, I branched off main instead of the feature branch